cmake/Hunter/config.cmake

```diff
@@ -33,3 +33,9 @@ hunter_config(
     SHA1 152b849610d91a9dfa1401293f43230c2e0c33f8
     CMAKE_ARGS BUILD_GMOCK=OFF gtest_force_shared_crt=ON
 )
+
+hunter_config(
+    libscrypt
+    VERSION 1.21-p1
+    CMAKE_ARGS CMAKE_C_FLAGS=-D_CRT_SECURE_NO_WARNINGS
```

Comment:
@chfast should it be better under `if (MSVC)` only?

Reply:
Yes.

Reply:
The whole `hunter_config()` can be inside `if(MSVC)` then.